server/scsynth/SC_PortAudio.cpp

```diff
@@ -25,6 +25,7 @@
 #include "SC_Time.hpp"
 #include <math.h>
 #include <stdlib.h>
+#include "../../common/SC_PaUtils.hpp"
```

Comment:
`common` is already an include directory so you just need to write `#include "SC_PaUtils.hpp"`